com.unity.robotics.urdf-importer/Runtime/UnityMeshImporter/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
```

Comment:
is it possible to fork AssimNet and MeshImporter repo and add them as a dependency instead of copy & paste all the files here?